ui/src/app/tasks/flo/metamodel.service.ts

```diff
@@ -206,7 +206,7 @@ export class MetamodelService implements Flo.Metamodel {
           metadata.set(this.COMPOSED_TASK_LABEL, element.attr('node-label'));
         }
         nodes.push(new Node(element.attributes.id, attrs.metadata.name, element.attributes.attrs.props, metadata));
-      } else if (element.get('type') === joint.shapes.flo.LINK_TYPE
+      } else if ((element.get('type') === joint.shapes.flo.LINK_TYPE || element.attributes.type === 'link')
```

Comment:
Hmm... was that the fix for graph - to - text? Don't think the type of the link should be anything else but ` joint.shapes.flo.LINK_TYPE`. Don't think that `'link'` should be there... wonder where it comes from. @jvalkeal please confirm if this is intentional change. If yes, I'll merge and check what lead to type being `'link'`